prow/jobs/kyma/docs/.ci/website-build-trigger.yaml

```diff
@@ -0,0 +1,64 @@
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+website_ref: &website_ref
+  org: kyma-project
+  repo: website
+  base_ref: master
+  path_alias: github.com/kyma-project/website
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    serviceAccountName: job-waiter
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/buildpack-golang:go1.11
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        args:
+          - "-c"
+          - "GITHUB_TOKEN=${BOT_GITHUB_TOKEN} /home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/job-guard.sh && cd /home/prow/go/src/github.com/kyma-project/kyma/docs/.ci && make ci-release"
+        env:
+          - name: JOB_FILTER_SUBSTRING
+            value: "github-release"
```

Comment:
This will not work :( :
 - Job Guard doesn't monitor statuses on master branch, only on PRs
 - Prow doesn't reports `postsubmit` jobs to GitHub - this can be enabled since February 13, 2019 https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md#announcements